### PR TITLE
Add Glazier themes

### PR DESCRIPTION
**Pull request by airgap**
1. Glazier | Phantom
2. Glazier | Deep Space
3. Glazier | Redrum

Early prototype

### CODE REVIEW

**Comment by airgap** [comment]
@cla-bot check

File: extensions.toml [hunk 1]
```diff
@@ -440,3 +440,7 @@ version = "0.0.1"
 submodule = "extensions/zed"
 path = "extensions/zig"
 version = "0.0.1"
+
+[glazier]
+submodule = "extensions/glazier"
+version = "0.0.0"
```
Comment (maxdeviant) [inline extensions.toml:446]: Could we use a non-zero version here?

**Review by maxdeviant — CHANGES_REQUESTED** [review]
Would you mind adjusting the version number so it doesn't have all zero components?

It looks like the Git submodule is missing as well (we have a `.gitmodules` entry, but no actual submodule committed).

**Review by maxdeviant — APPROVED** [review]
@airgap Thank you!

**Pull request merged**